istio-operator/istio-operator.spec

```diff
@@ -109,12 +109,27 @@ popd
 install -d $RPM_BUILD_ROOT/manifests
 cp -ra OPERATOR/src/github.com/maistra/istio-operator/manifests-maistra/* $RPM_BUILD_ROOT/manifests
 
+# install the templates
+TEMPLATES_DIR=$RPM_BUILD_ROOT%{_datadir}/istio-operator/default-templates
+mkdir -p ${TEMPLATES_DIR}
+if [[ %{community_build}  == "true" ]]; then
```

Comment:
Can you surround ` %{community_build}` with quotes?

Reply:
Done